tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Elasticsearch/ElasticsearchTags.cs

```diff
@@ -24,5 +25,36 @@ internal partial class ElasticsearchTags : InstrumentationTags
 
         [Tag(Trace.Tags.ElasticsearchUrl)]
         public string Url { get; set; }
+
+        [Tag(Trace.Tags.OutHost)]
+        public string Host { get; set; }
```

Comment:
I would have expected changes in metadata files

Reply:
Both `tracer/test/Datadog.Trace.TestHelpers/SpanMetadataV0Rules.cs` and `tracer/test/Datadog.Trace.TestHelpers/SpanMetadataV1Rules.cs` have been updated

Reply:
Ohh I didn't update the markdown file. That's a manual update at the moment, but we should add this to our set of source-generator GitHub actions checks later. I'll update the markdown file manually right now

Reply:
Updated in 7d1d9eab9895571c60262e5154867f52006897ca